src/action.js

```diff
@@ -155,10 +119,8 @@ class Action extends Emitter {
   /**
    * Set up an action such that it depends on the result of another
    * series of actions.
-   * @param {Array.<Action>} actions
-   * @return {this}
    */
-  link(actions) {
+  link(actions: Action[]): this {
```

Comment:
is this the same as `Array<Actions>`...it is...cool TIL. This is shorter I should change my stuff to be like this. But `Array<Actions>` reads better to me. When we talk about how we will eventually do flow we should point this out and chose a standard.